src/components/Nav/nav.js

```diff
@@ -1,108 +1,118 @@
-import React from 'react';
-import './nav.css';
-import Logo from '../Logo/Logo';
-import $ from 'jquery';
+import React from "react";
+import "./nav.css";
+import Logo from "../Logo/Logo";
+import $ from "jquery";
 
-// export default function Nav() {
+// eslint-disable-next-line react/prop-types
```

Comment:
This is how you disable checks. We'll prob need to deal with this at some point or disable it globally.

Reply:
Added to documentation for #58